OpenRA.Game/ModMetadata.cs

```diff
@@ -24,6 +24,7 @@ public class ModMetadata
 		public string Version;
 		public string Author;
 		public bool Hidden;
+		public ContentInstaller AssetInformation;
```

Comment:
Can we call the variable `Installer` or something else more closely related to the type?